src/gui/guiHyperText.cpp

```diff
@@ -423,40 +423,65 @@ u32 ParsedText::parseTag(const wchar_t *text, u32 cursor)
 		std::string attr_name = "";
 		core::stringw attr_val = L"";
 
+		// Consume whitespace
 		while (c == ' ') {
 			c = text[++cursor];
 			if (c == L'\0' || c == L'=')
 				return 0;
 		}
 
+		// Read attribute name
 		while (c != L' ' && c != L'=') {
 			attr_name += (char)c;
 			c = text[++cursor];
 			if (c == L'\0' || c == L'>')
 				return 0;
 		}
 
+		// Consume whitespace
 		while (c == L' ') {
 			c = text[++cursor];
 			if (c == L'\0' || c == L'>')
 				return 0;
 		}
 
+		// Skip equals
 		if (c != L'=')
 			return 0;
-
 		c = text[++cursor];
-
 		if (c == L'\0')
 			return 0;
 
-		while (c != L'>' && c != L' ') {
-			attr_val += c;
+		// Read optional quote
+		wchar_t quote_used;
+		if (c == L'"' || c == L'\'') {
+			quote_used = c;
 			c = text[++cursor];
 			if (c == L'\0')
 				return 0;
 		}
 
+		// Read attribute value
+		bool escape = false;
+		while (escape || (quote_used && c != quote_used) || (!quote_used && c != L'>' && c != L' ')) {
```

Comment:
can be simplified:
```suggestion
		while (escape || (c != quote_used && c != L'>' && c != L' ')) {
```

with your PR it also becomes possible to do e.g. `<tag attr=hello\ world>`, is that intentional?

Reply:
Wouldn't this suggestion require escaping `>` and the whitespace even in quoted text? Otherwise `quote_used ? (c != quote_used) : (c != L'>' && c != L' ')` would IMO make sense.

Reply:
I was aware that this is possible, it's a side effect of this change respecting escape characters. It would take explicit code to not allow this. Is there a reason to not allow it? I guess it's not actually documented and people should prefer quoting

Reply:
I don't with an issue with this per se, just someone might rely on it when it's not intended.